libafl_frida/src/coverage_rt.rs

```diff
@@ -0,0 +1,177 @@
+use dynasmrt::{dynasm, DynasmApi, DynasmLabelApi};
+use std::ffi::c_void;
+
+#[cfg(target_arch = "x86_64")]
+use frida_gum::instruction_writer::X86Register;
+#[cfg(target_arch = "aarch64")]
+use frida_gum::instruction_writer::{Aarch64Register, IndexMode};
+
+use frida_gum::{instruction_writer::InstructionWriter, stalker::StalkerOutput};
+
+/// (Default) map size for frida coverage reporting
+pub const MAP_SIZE: usize = 64 * 1024;
+
+pub struct CoverageRuntime {
+    map: [u8; MAP_SIZE],
+    previous_pc: [u64; 1],
```

Comment:
Why is this an array?

Reply:
it was an array before ;)
but i'll fix

Reply:
Ok. Didn't realize... Yeah if you can fix...